### PR TITLE
DOC: Make explicit that create-sibling-git(hub|lab) are optional

### PR DESCRIPTION
**Pull request by mih**
...but useful. This addresses some users' assumption that any such
linkage needs to be established through these commands, even if
there already is a corresponding project available.

Fixes gh-4120

### CODE REVIEW

File: datalad/distribution/create_sibling_github.py [hunk 1]
```diff
@@ -53,11 +53,13 @@
...
-    A repository can be created under a user's Github account, or any
-    organization a user is a member of (given appropriate permissions).
-
-    Recursive sibling creation for subdatasets is supported. A dataset
-    hierarchy is represented as a flat list of Github repositories.
+    An existing GitHub project, or a project created via the GitHub website can
+    be configured as a sibling with the :command:`siblings` command.
+    Alternatively, this command can create a repository under a user's Github
+    account, or any organization a user is a member of (given appropriate
+    permissions).  This is particulary helpful for recursive sibling creation
```
Comment (adswa) [inline datalad/distribution/create_sibling_github.py]:
```suggestion
    permissions). This is particulary helpful for recursive sibling creation
```

File: datalad/distribution/create_sibling_github.py [hunk 2]
```diff
@@ -53,11 +53,13 @@
...
+    for subdatasets. In such a case, a dataset hierarchy is represented as a
+    flat list of Github repositories.
```
Comment (adswa) [inline datalad/distribution/create_sibling_github.py]:
```suggestion
    flat list of GitHub repositories.
```

File: datalad/distributed/create_sibling_gitlab.py [hunk 3]
```diff
@@ -56,12 +56,16 @@
...
-    appropriate permissions for. API access and authentication are implemented
-    via python-gitlab, and all its features are supported. A particular GitLab
-    site must be configured in a named section of a python-gitlab.cfg file
-    (see https://python-gitlab.readthedocs.io/en/stable/cli.html#configuration
-    for details), such as::
+    An existing GitLab project, or a project created via the GitLab web
+    interface can be configured as a sibling with the :command:`siblings`
+    command. Alternativly, this command can create a GitLab project at any
+    location/path a given user has appropriate permissions for. This is
+    particulary helpful for recursive sibling creation for subdatasets.  API
```
Comment (adswa) [inline datalad/distributed/create_sibling_gitlab.py]:
```suggestion
    particulary helpful for recursive sibling creation for subdatasets. API
```

**Comment by mih** [comment]
I will cancel the CIs, only whitespace change.

Thx @adswa for the review.

**Pull request merged**